support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -12,6 +12,8 @@ import type { Tests } from './abtest';
 const allLandingPagesAndThankyouPages = '/contribute|thankyou(/.*)?$';
 const notUkLandingPage = '/us|au|eu|int|nz|ca/contribute(/.*)?$';
 export const subsShowcaseAndDigiSubPages = '(/??/subscribe(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)';
+export const digiSubLandingPages = '(/??/subscribe/digital/gift(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)';
```

Comment:
I think this is unused?

Reply:
I didn't want to delete it because we're almost certainly going to need it for other tests this quarter. It might make more sense to put these regexes inside an object instead of having them all as separate variables though.